graylog2-web-interface/src/components/users/EditRolesForm.jsx

```diff
@@ -39,11 +43,22 @@ const EditRolesForm = React.createClass({
   _onCancel() {
     this.props.history.pushState(null, Routes.SYSTEM.AUTHENTICATION.USERS.LIST);
   },
+  _onValueChange(newRoles) {
+    const roles = newRoles.split(',');
+    this.setState({ newRoles: roles });
+  },
   render() {
     const user = this.props.user;
     if (!this.state.roles) {
       return <Spinner />;
     }
+    let rolesAlert = null;
+    const roles = this.state.newRoles;
+    if (roles != null && roles.indexOf('Reader') === -1 && roles.indexOf('Admin') === -1) {
+      rolesAlert = <Alert bsStyle="danger" role="alert" className={EditRolesFormStyle.rolesMissingAlert}>
```

Comment:
Small nitpick: this triggers a [`react/wrap-multilines`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-wrap-multilines.md) styling warning.